api/envoy/admin/v2alpha/clusters.proto

```diff
@@ -43,6 +43,19 @@ message ClusterStatus {
 
   // Mapping from host address to the host's current status.
   repeated HostStatus host_statuses = 4;
+
+  // The success rate threshold used in the last interval for locally originated failures.
+  // The threshold is used to eject hosts based on their success rate. See
+  // :ref:`Cluster outlier detection <arch_overview_outlier_detection>` statistics
```

Comment:
nit: end sentence with period. Can you fix elsewhere in this section if you see similar instances)

Reply:
Per above, maybe reference `split_external_local_origin_errors` and say that this field won't do anything if that is not true?